apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
We inline the [`IN_STORYBOOK`](https://github.com/code-dot-org/code-dot-org/blob/8cded4767e60432423e18be6c0cdf9a64d185250/apps/webpack.js#L141-L141) constant, that'd be a good place to start.